src/widgets/index/index.ts

```diff
@@ -25,6 +25,82 @@ import {
   mergeSearchParameters,
 } from '../../lib/utils';
 
+import { WidgetType, WidgetParams, Schema } from '../../../schema';
+
+function Pbf() {
```

Comment:
I like that we didn't need to use a dependency here, but I'd probably prefer it in its own file, and with tests

Reply:
Agree, but for now in this POC I just made it "works" :wink:, then is still tons of improvement in quality/structure/testing to do.